### PR TITLE
Completion of the stage of translation of the course into Russian

### PR DESCRIPTION
**Pull request by blademoon**
Finalizing the translation phase. Units 6 and 7 have been translated. #109

### CODE REVIEW

File: chapters/ru/chapter7/speech-to-speech.mdx [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+```python
+def translate(audio):
+    outputs = pipe(audio, max_new_tokens=256, generate_kwargs={"task": "translate"})
+    return outputs["text"]
+```
+
+<Tip>
+
+    Whisper также можно "обманом" перевести с речи на любом языке X на любой язык Y. Просто задайте задачу `"transcribe"`, а `"language"` - целевым языком
```
Comment (MKhalusova) [inline chapters/ru/chapter7/speech-to-speech.mdx:91]: Looks like a verb is missing. "обманом" (заставить?) перевести

File: chapters/ru/chapter7/speech-to-speech.mdx [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    в аргументах ключевых слов генерации, например, для испанского языка можно задать:
+
+	`generate_kwargs={"task": "transcribe", "language": "es"&rcub;`
```
Comment (MKhalusova) [inline chapters/ru/chapter7/speech-to-speech.mdx:94]:
```suggestion
	`generate_kwargs={"task": "transcribe", "language": "es"}`
```

File: chapters/ru/chapter7/speech-to-speech.mdx [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+Хорошо! Если мы сравним это с исходным текстом:
+
+```python
+sample["raw_text"]
+```
+```
+'Penso che questo sia un passo in avanti importante nella costruzione di uno spazio giuridico di libertà di circolazione e di protezione dei diritti per le persone in Europa.'
+```
+
+Мы увидим, что перевод более или менее совпадает (вы можете проверить это с помощью Google Translate), за исключением небольшого
```
Comment (MKhalusova) [inline chapters/ru/chapter7/speech-to-speech.mdx:116]:
```suggestion
Мы увидим, что перевод более или менее совпадает (вы можете проверить это с помощью Google Translate), за исключением нескольких
```

File: chapters/ru/chapter7/speech-to-speech.mdx [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+```python
+def synthesise(text):
+    inputs = processor(text=text, return_tensors="pt")
+    speech = model.generate_speech(
+        inputs["input_ids"].to(device), speaker_embeddings.to(device), vocoder=vocoder
+    )
+    return speech.cpu()
+```
+
+Let's check it works with a dummy text input:
```
Comment (MKhalusova) [inline chapters/ru/chapter7/speech-to-speech.mdx:175]: This sentence hasn't been translated.

**Review by MKhalusova — CHANGES_REQUESTED** [review]
Massive kudos for working on the translation! I found a couple of minor things to fix, after that it should be ready to merge once [chapter 5](https://github.com/huggingface/audio-transformers-course/pull/123) is merged.

File: chapters/ru/chapter6/pre-trained_models.mdx [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+vocoder = SpeechT5HifiGan.from_pretrained("microsoft/speecht5_hifigan")
+```
+
+Теперь достаточно передать его в качестве аргумента при генерации речи, и выходные данные будут автоматически преобразованы в форму волны речи.
+
+```python
+speech = model.generate_speech(inputs["input_ids"], speaker_embeddings, vocoder=vocoder)
+```
+
+Let's listen to the result. The sample rate used by SpeechT5 is always 16 kHz.
```
[the remark this answers is not in the document]
Comment (blademoon) [reply chapters/ru/chapter6/pre-trained_models.mdx]: Yes, indeed, corrected.

[the remark this answers is not in the document]
Comment (blademoon) [reply chapters/ru/chapter6/pre-trained_models.mdx]: Yes, indeed, corrected.

File: chapters/ru/chapter7/speech-to-speech.mdx (hunk 1 above)
Comment (blademoon) [reply chapters/ru/chapter7/speech-to-speech.mdx]: Corrected.

File: chapters/ru/chapter7/speech-to-speech.mdx (hunk 2 above)
Comment (blademoon) [reply chapters/ru/chapter7/speech-to-speech.mdx]: Corrected)

File: chapters/ru/chapter7/speech-to-speech.mdx (hunk 3 above)
Comment (blademoon) [reply chapters/ru/chapter7/speech-to-speech.mdx]: Done.

File: chapters/ru/chapter7/speech-to-speech.mdx (hunk 4 above)
Comment (blademoon) [reply chapters/ru/chapter7/speech-to-speech.mdx]: Added a translation of the missing sentence. Thank you for your attention Maria.

**Comment by blademoon** [comment]
@MKhalusova :rocket: Поехали)))

**Review by MKhalusova — COMMENTED** [review]
Overall great translation! Thank you for working on it. I have left a few minor nits.
We have also recently updated the TTS section with Bark, so if you'd like to include it in the translation as well, that would be awesome!

File: chapters/ru/chapter6/pre-trained_models.mdx [hunk 6]
```diff
@@ -0,0 +1,258 @@
+# Предварительно обученные модели text-to-speech
```
[the remark this answers is not in the document]
Comment (blademoon) [reply chapters/ru/chapter6/pre-trained_models.mdx:1]: Thanks Maria. Let's continue the discussion in  our issues [#109](https://github.com/huggingface/audio-transformers-course/issues/109).

[the remark this answers is not in the document]
Comment (blademoon) [reply chapters/ru/chapter6/pre-trained_models.mdx:1]: Let's take this translation all the way to the end. Then we will add new material.

File: chapters/ru/chapter6/pre-trained_models.mdx [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+предварительно обученная контрольная точка поддерживает только английский язык:
+
+<iframe
+	src="https://matthijs-speecht5-tts-demo.hf.space"
+	frameborder="0"
+	width="850"
+	height="450">
+</iframe>
+
+## Массивная многоязыковая речь (MMS)
```
[the remark this answers is not in the document]
Comment (blademoon) [reply chapters/ru/chapter6/pre-trained_models.mdx:184]: I think you're right. Let's leave the model name untranslated.

File: chapters/ru/chapter6/pre-trained_models.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (MKhalusova) [reply chapters/ru/chapter6/pre-trained_models.mdx:1]: Sure, feel free to add Bark in a separate PR

**Review by MKhalusova — APPROVED** [review]

**Pull request merged**